src/WebJobs.Extensions.DurableTask/DurableTaskExtension.cs

```diff
@@ -710,15 +704,15 @@ internal void RegisterActivity(FunctionName activityFunction, ITriggeredFunction
             }
             else
             {
+                var info = new RegisteredFunctionInfo(executor, isOutOfProc: false);
+                this.knownActivities[activityFunction] = info;
+
                 this.TraceHelper.ExtensionInformationalEvent(
                     this.Options.HubName,
                     instanceId: string.Empty,
                     functionName: activityFunction.Name,
-                    message: $"Registering activity function named {activityFunction}.",
+                    message: $"Registered activity function named {activityFunction}.",

```

Comment:
These changes were simply to make it easier for me to filter trace messages by making them all consistent (starting messages with "Registered" instead of having some of them start with "Registering"). There is no functional change here.